index.bs

```diff
@@ -1388,6 +1389,12 @@ that are returned to the caller when a new credential is created, or a new asser
         {{CredentialsContainer/create()}}, this attribute's value will be an {{AuthenticatorAttestationResponse}}, otherwise,
         the {{PublicKeyCredential}} was created in response to {{CredentialsContainer/get()}}, and this attribute's value
         will be an {{AuthenticatorAssertionResponse}}.
+    
+    :   <dfn>authenticatorAttachment</dfn>
+    ::  This attribute contains the authenticatorAttachment used to communicate to the [=authenticator=], or `null` if 
```

Comment:
Can you make authenticatorAttachment a reference to the AuthenticatorAttachment enum? I believe

```suggestion
    ::  This attribute contains the {{AuthenticatorAttachment}} used to communicate to the [=authenticator=], or `null` if 
```

should do the trick.

Reply:
This should also incorporate a "The value SHOULD be a member of AuthenticatorAttachment" part like other enum-ish fields. I'm not sure what the recommendation should be for unknown values, but maybe something like:

```
The value SHOULD be a member of {{AuthenticatorAttachment}}, and Relying Parties SHOULD treat unknown values as if the value were null.
```